gui/wxpython/location_wizard/wizard.py

```diff
@@ -1486,41 +1481,23 @@ def OnEnterPage(self, event):
         event.Skip()
 
     def OnPageChanging(self, event):
-        if event.GetDirection() and not os.path.isfile(self.wktfile):
```

Comment:
There needs to be a condition, you are trying to prevent from going to next page if there is no text (previously - if the file doesn't exist). Ideally it would check the WKT string is valid, but that's very difficult, so we will just check for any string. It's technically not needed because you already deal with that by disabling the next button, but let's keep it there.

 I was surprised your current code works since you don't have any condition there (so it would prevent the page from switching every time), but that's because of the event.Skip() - you need to remove it from there. Please also remove event Skip from GeoreferencedFilePage (the same function), it's the same case, it shouldn't be there.